apps/web/src/App.tsx

```diff
@@ -71,6 +72,7 @@ function App() {
     <HelmetProvider>
       <BrowserRouter basename={CONTEXT_PATH}>
         <QueryClientProvider client={queryClient}>
+          <ReactQueryDevtools />
```

Comment:
Can we only show this in dev environment?

Reply:
sorry I forgot to push the last commit :D